metadata-ingestion/tests/integration/postgres/postgres_mces_with_db_golden.json

```diff
@@ -351,65 +351,7 @@
             },
             "rowCount": 2,
             "columnCount": 9,
-            "fieldProfiles": [
```

Comment:
just confirming that this change is actually correct

kinda hard to understand the correspondence between the yml files and the golden files - they really should've been named better

Reply:
Yup, this is what no fields profiled looks like